# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../shorewall/makefile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- shorewall-1.4.6b.orig/Makefile	1969-12-31 19:00:00.000000000 -0500
+++ shorewall-1.4.6b/Makefile	2003-08-18 11:13:53.000000000 -0400
@@ -0,0 +1,4 @@
+all:
+install:
+	./install.sh /etc/rc.d/init.d
+	
